### ISSUE TITLE
mining point not registering gusgen mines

### ISSUE THREAD

**Issue by Ohki0523** [EdenServer/community#1716]
version 30181205_0

the mining point in the tunnel at (F-7) isn't allowing me to mine it unless i logout and back in.  I've been skipping this point so I don't know when it started. I can target it and attempt to use a pickaxe on it, however the animation for mining doesn't happen and I get 0 results.  Logout then back in fixes the issue until I circle back around to this one.  No other spots seem to have it

### Checklist

- [X ] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [X ] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [ X] I have provided my client version in the details. (type /ver into your game window)

**Comment by Setz0r** [comment]
Makes me wonder if the point(NPC) is loaded but not registered as a loaded(minable) point in the helm system.  Could be a client side issue with entity updates as well.

**Comment by burkewitz** [comment]
It’s not just that point, and it’s a widely known issue among miners. As I dualbox, I will tell you it is 100% a client side issue. A fake node will appear on one and not the other character, and the fake node will clear when it’s eventually populated with a real one and is mined by the character it’s bugged out on. Sucks but the solution at the moment is to keep mining like normal and keep checking the node

**Comment by Setz0r** [comment]
The problem might be that the points don't update zonewide, so if one pops while you're close to it, then someone else mines it while you're out of range, when you come back into range, your client still thinks it's loaded.

**Comment by burkewitz** [comment]
Yup. The problem does eventually solve itself. At worst is a miner (LAWL) inconvenience causing you to check what might be a bum node every time you make a pass.

**Comment by xizeta** [comment]
I've been mining for some time now and have only encountered this issue in Gusgen Mines.

**Comment by Setz0r** [comment]
wonder if not all the mining point npcs are setup in the helm script

**Comment by Fng098** [comment]
I am having a similar issue in Zeruhn Mines.

https://youtu.be/_OhyyXkA_94
